tests/v0_tests/test_marqo_cloud_instance_mapping.py

```diff
@@ -121,6 +121,17 @@ def test_request_of_not_existing_index_raises_error(self, mock_get):
         with self.assertRaises(MarqoCloudIndexNotFoundError):
             mapping.get_index_base_url("index2")
 
+    @patch("requests.get")
+    def test_modifying_state_returns_as_ready(self, mock_get):
```

Comment:
Please test all five cloud states (creating, ready, modifying, deleting, deleted) are handled appropriately. This covers modifying so four more to test.